src/cheri/insns/cbo_exceptions.adoc

```diff
@@ -11,7 +11,9 @@ ifdef::cbo_clean_flush[]
 endif::cbo_clean_flush[]
 
 ifdef::cbo_inval[]
-| {cheri_excep_name_st} | {cheri_excep_desc_perm} <<w_perm>>, <<r_perm>> and <<asr_perm>> are all required.
+| {cheri_excep_name_st} | {cheri_excep_desc_perm} <<w_perm>>, <<r_perm>> are both required.
+
+If the PMA is _CHERI Valid Tag_ and CBO.INVAL is not executing as CBO.FLUSH then <<asr_perm>> is required in the <<pcc>>.
```

Comment:
Doesn't checking the PMA now introduce a dependency on the address that is being accessed? I'd just keep it simple and always require PCC.ASR?

Reply:
I'm less concerned about the CBO.INVAL->CBO.FLUSH mapping since that is just checking a single CSR bit, but the first one does not make sense to me.

Reply:
ok - the point was that if it's a FLUSH it's not a problem (in general) and if the PMA doesn't support tags then it's no worse then in non-CHERI

but OK - I'm trying to avoid connecting PCC.ASR further than it needs to be connected. We can also solve this by changing the fault type to `CHERI Instruction Access Fault` and then the fault can be raised in the decoder - it's a PCC fault after all.